fe/fe-core/src/main/java/com/starrocks/statistic/AnalyzeManager.java

```diff
@@ -48,6 +50,9 @@ public class AnalyzeManager implements Writable {
     private final Map<Pair<Long, String>, HistogramStatsMeta> histogramStatsMetaMap;
     //ConnectContext of all currently running analyze tasks
     private final Map<Long, ConnectContext> connectionMap = Maps.newConcurrentMap();
+    private static final ExecutorService ANALYZE_TASK_THREAD_POOL = ThreadPoolManager.newDaemonFixedThreadPool(
+            Config.statistic_collect_concurrency, Config.statistic_collect_concurrency * 3,
```

Comment:
The queue size is too small, if one user want to analyze all tables in one database, he will get the  error message.